app/views/RoomView/Header/Header.js

```diff
@@ -106,7 +108,7 @@ const Header = React.memo(({
 	}
 
 	return (
-		<View style={[styles.container, { width: width - widthOffset }]}>
+		<View style={[styles.container, { width: width - (width * tmid ? 0.2 : 0.3) }]}>
```

Comment:
Why did you remove `widthOffset`?
What `width * tmid` does?

This way, when `unread` is null, `title` is not left aligned nor centered.
<img src="https://user-images.githubusercontent.com/804994/63284557-38755600-c28a-11e9-95a6-d8e86011beee.png" width="350" />


This is what widthOffset does:
<img src="https://user-images.githubusercontent.com/804994/63284520-2a273a00-c28a-11e9-9722-b238bdfc06be.png" width="350" />